shared/components/BaseButton/BaseButton.modules.scss

```diff
@@ -1,5 +1,5 @@
 @import '~@tds/core-colours/colours';
-@import '../../scss/responsive';
+@import '../../../packages/Responsive/scss/responsive.scss';
```

Comment:
This should import responsive.scss from the core-responsive package, so that core-responsive is declared as a dependency. Notice line 1.

It should be something like `@import '~@tds/core-responsive/responsive';`